pkg/recorder/ffmpeg.go

```
@@ -0,0 +1,81 @@
+		args = append(args, strings.Split(r.AppCnf.FfmpegSettings.Recording.PreInput, " ")...)
+		args = append(args,
+			"-video_size", fmt.Sprintf("%dx%d", r.AppCnf.Recorder.Width, r.AppCnf.Recorder.Height),
+			"-f", "x11grab",
+			"-i", r.displayId,
+			"-f", "pulse",
+			"-i", fmt.Sprintf("%s.monitor", r.pulseSinkName),
+		)
+		args = append(args, strings.Split(r.AppCnf.FfmpegSettings.Recording.PostInput, " ")...)
+		args = append(args, mp4File)
+	} else {
+		return errors.New(fmt.Sprintf("invalid task %s received", r.Req.Task.String()))
```

golangci-lint (1.62.2)

S1028: should use fmt.Errorf(...) instead of errors.New(fmt.Sprintf(...))

(gosimple)